blocks/library/gallery/index.js

```diff
@@ -133,7 +133,7 @@ registerBlockType( 'core/gallery', {
 	save( { attributes } ) {
 		const { images, columns = defaultColumnsNumber( attributes ), align, imageCrop, linkTo } = attributes;
 		return (
-			<div className={ `align${ align } columns-${ columns } ${ imageCrop ? 'is-cropped' : '' }` } >
+			<ul className={ `align${ align } columns-${ columns } ${ imageCrop ? 'is-cropped' : '' }` } >
```

Comment:
Can't review fully today but quick note here, we should add a "deprecated" section (see quote block as an example) to avoid invalidating existing galleries.